src/components/Form/Form.mdx

```diff
@@ -27,22 +26,15 @@ import { changelogData } from "./formChangelogData";
 
 ## Table of Contents
 
-- {<Link href="#overview" target="_self">Overview</Link>}
-- {<Link href="#component-props" target="_self">Component Props</Link>}
-- {<Link href="#accessibility" target="_self">Accessibility</Link>}
-- {<Link href="#spacing-variants" target="_self">Spacing Variants</Link>}
-- {<Link href="#example-code" target="_self">Example Code</Link>}
-- {<Link href="#changelog" target="_self">Changelog</Link>}
+- [Overview](#overview)
+- [Component Props](#component-props)
+- [Accessibility](#accessibility)
+- [Spacing Variants](#spacing-variants)
+- [Example Code](#example-code)
+- [Changelog](#changelog)
 
 ## Overview
 
-<Description of={FormStories} />
-
-The `Form` component renders a standard `<form>` element and should be used to
-handle layout and spacing for child input fields. `FormRow` and `FormField`
-components should be used to build the `<form>` structure and to arrange input
-fields as needed.
```

Comment:
What made you remove this?

Reply:
Hmm not sure. I don't think I meant to remove this.